debian/postinst

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+set -e
+
+#DEBHELPER#
+
+PACKAGE_NAME="schaufel"
+PACKAGE_USER="schaufel"
+PACKAGE_GROUP="schaufel"
+CONF_DIR="/etc/${PACKAGE_NAME}"
+WORK_DIR="/var/lib/${PACKAGE_NAME}"
+LOG_DIR="/var/log/${PACKAGE_NAME}"
+
+## Create group if not exist
+if ! getent group | grep --quiet "^${PACKAGE_GROUP}:" ; then
+    echo -n "Adding system group ${PACKAGE_GROUP} .."
+    addgroup --quiet --system ${PACKAGE_GROUP} 2>/dev/null || true
+    echo ".. done"
+fi
+
+## Create empty home directory if not exist
+test -d ${WORK_DIR} || mkdir -p ${WORK_DIR}
+
+## Create user if not exist
+if ! getent passwd | grep --quiet "^${PACKAGE_USER}:" ; then
+    echo -n "Adding system user ${PACKAGE_USER} .."
+    adduser --quiet --system \
```

Comment:
same here about UID, somehow it is like that:
https://github.com/adjust/gentoo-overlay/blob/7bad862bcae5a057e596f15f0b6a24ab06e1f428/acct-user/schaufel/schaufel-0.ebuild#L10